### PR TITLE
PMM-4926 Add Stats for connection checker.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-452/

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -160,8 +160,14 @@ message CheckConnectionRequest {
 
 // CheckConnectionResponse is an AgentMessage containing a result of connection check.
 message CheckConnectionResponse {
+  // Stats contains various Service statistics.
+  message Stats {
+    // A number of tables. 0 means "unknown" (for compatibility), -1 means "there are really 0 tables".
```
Comment (BupycHuk) [inline api/agentpb/agent.proto]: let's use extra boolean field instead of using -1

Comment (AlekSi) [reply api/agentpb/agent.proto]: Actually, it is not even possible: `SELECT COUNT(*) FROM information_schema.tables` always returns non-zero value – `information_schema.tables` is a table itself. 😆

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-452/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-452/

**Pull request merged**